lib/dal/tests/integration_test/validations.rs

```diff
@@ -230,91 +229,91 @@ async fn validation_pre_post_mgmt_func(ctx: &mut DalContext) -> Result<()> {
     Ok(())
 }
 
-#[ignore]
-#[test]
-async fn validation_on_dependent_value(ctx: &mut DalContext) -> Result<()> {
-    let output_component =
-        create_component_for_default_schema_name_in_default_view(ctx, "ValidatedOutput", "Output")
-            .await?;
-    let input_component =
-        create_component_for_default_schema_name_in_default_view(ctx, "ValidatedInput", "Input")
-            .await?;
-
-    ChangeSetTestHelpers::commit_and_update_snapshot_to_visibility(ctx).await?;
-
-    connect_components_with_socket_names(
-        ctx,
-        output_component.id(),
-        "number",
-        input_component.id(),
-        "number",
-    )
-    .await?;
-
-    let prop_path = &["root", "domain", "a_number"];
-    let av_id = output_component
-        .attribute_values_for_prop(ctx, prop_path)
-        .await?
-        .pop()
-        .expect("there should only be one value id");
-
-    AttributeValue::update(ctx, av_id, Some(json!(1))).await?;
-
-    ChangeSetTestHelpers::commit_and_update_snapshot_to_visibility(ctx).await?;
-
-    let source_prop_view = PropEditorTestView::for_component_id(ctx, output_component.id())
-        .await?
-        .get_value(prop_path)?;
-    let destination_prop_view = PropEditorTestView::for_component_id(ctx, input_component.id())
-        .await?
-        .get_value(prop_path)?;
-
-    // Check validations and values
-    let source_result = extract_value_and_validation(source_prop_view)?;
-    assert_eq!(
-        json!({
-            "value": 1,
-            "validation": {
-                "status": "Success",
-                "message": null
-            }
-
-        }),
-        source_result
-    );
-
-    let destination_result = extract_value_and_validation(destination_prop_view)?;
-    assert_eq!(source_result, destination_result,);
-
-    AttributeValue::update(ctx, av_id, Some(json!(3))).await?;
-
-    ChangeSetTestHelpers::commit_and_update_snapshot_to_visibility(ctx).await?;
-
-    let source_prop_view = PropEditorTestView::for_component_id(ctx, input_component.id())
-        .await?
-        .get_value(prop_path)?;
-    let destination_prop_view = PropEditorTestView::for_component_id(ctx, output_component.id())
-        .await?
-        .get_value(prop_path)?;
-
-    let source_result = extract_value_and_validation(source_prop_view)?;
-    assert_eq!(
-        json!({
-            "value": 3,
-            "validation": {
-                "status": "Failure",
-                "message": "\"value\" must be less than or equal to 2"
-            }
-
-        }),
-        source_result
-    );
-
-    let destination_result = extract_value_and_validation(destination_prop_view)?;
-    assert_eq!(source_result, destination_result,);
-
-    Ok(())
-}
+// #[ignore]
```

Comment:
Was ignored; now doesn't compile because connections. Probably could be updated to use subscriptions, but may as well let that happen when it actually gets unignored.